charts/c100-application/templates/sidekiq.yaml

```diff
@@ -46,9 +46,13 @@ spec:
             preStop:
               exec:
                 command: ["./sidekiq.sh", "stop"]
-          # non-secret env vars defined in `config_map.yaml`
+          {{ if .Values.global.enableKeyVaults }}
```

Comment:
do you need this if?

Reply:
If not, the dev environment tries to build a keyvault and fails.